### PR TITLE
Display all the events on a heatmap-style calendar

### PR DESCRIPTION
**Pull request by aguzovatii**
This pr adds a heatmap-style calendar where all the events that are fetched from the backend are displayed:
image

### CODE REVIEW

**Comment by Guzovatii-L** [comment]
wait, please, so I can understand what is happening!!

File: app/calendar-heatmap.tsx
Comment (Guzovatii-L) [inline app/calendar-heatmap.tsx:1]: hey, on the 21 line of code, what's happening if value.count is greater than 4? based on app/global.css

**Review by Guzovatii-L — COMMENTED** [review]
you have a bug

File: app/calendar-heatmap.tsx
Comment (aguzovatii) [reply app/calendar-heatmap.tsx:1]: Fixed in a new commit, please try again

**Review by Guzovatii-L — APPROVED** [review]

File: app/page.tsx [hunk 1]
```diff
@@ -1,47 +1,77 @@
...
+  const calendar = await getCalendar()
+  console.log(calendar)
+  for(var event of calendar.events) {
+    event.date_time = new Date(event.date_time);
   }
 
-  return res.json()
-}
+  let values = [];
+  for (var d = new Date(startDate); d <= endDate; d.setDate(d.getDate() + 1)) {
```
Comment (DianaJacota) [inline app/page.tsx:21]: Couldn't this for in for be improved?

**Review by DianaJacota — APPROVED** [review]
It looks good. Thank you for the cleanup🍪

File: app/page.tsx (hunk 1 above)
Comment (aguzovatii) [reply app/page.tsx:21]: Yes, you are right, we can do better.
I added a new task on our todo list, for now the current version is good enough

**Pull request merged**